readme.md

```diff
@@ -15,14 +19,54 @@
+- [TechWorldwithNana](https://www.youtube.com/c/TechWorldwithNana) Everything about technologies
+- [EngineerMan](https://www.youtube.com/c/EngineerMan) Everything about tech, hands on code examples. Interesting use cases.
+- [thenewboston](https://www.youtube.com/user/thenewboston) Around many technologies, frameworks and languages.
+
+
+## Learning-Platforms
+- [ACloudGuru](https://acloudguru.com/) - Cloud, K8s, Linux. Sandbox environments is a plus.
+
+## Books
+- [Designing Data intensive applications](https://www.amazon.pl/Designing-Data-Intensive-Applications-Reliable-Maintainable/dp/1449373321) - Great source of internals of distributed systems. Written By Martin Kleppmann, also reccomend his. Transactions, replicas, partitioning, data streaming and more interesting topics.
+- [Data management At Scale](https://www.amazon.pl/Data-Management-Scale-Enterprise-Architecture/dp/149205478X) - Good book about managing data in organization, small references about Data Mesh paradigm and avoding Data silos.
+- [Streaming systems](https://www.amazon.pl/Streaming-Systems-Where-Large-Scale-Processing/dp/1491983876) - Created by Tyler Akidau, PMC member of Apache Beam project, one of the must read books for stream processing people.
+- [Stream processing with apache Flink](https://www.amazon.pl/Stream-Processing-Apache-Flink-Implementation/dp/149197429X) - Created by coauthors of Apache Flink, Flink APIs maybe are a little bit stale but still worth to read about stream processing (great intro) and also Apache Flink internals.
+- [Learning Spark 2nd edition](https://www.amazon.pl/Learning-Spark-Lightning-fast-Data-Analytics/dp/1492050040) - Great book to read more about Apache spark
+- [Apache Kafka the definitive guide](https://www.confluent.io/resources/kafka-the-definitive-guide-v2/) You can get the book for free, it's everything about Apache Kafka
+- [Building Event-Driven Microservices: Leveraging Organizational Data at Scale](https://www.amazon.pl/Building-Event-Driven-Microservices-Leveraging-Organizational/dp/1492057894). Great book about creating event driven applications/microservices.
+- [Data Mesh](https://www.amazon.pl/Data-Mesh-Delivering-Data-Driven-Value/dp/1492092398) - Book written by author of Data Mesh paradigm. 
+- [Foundations of Scalable Systems: Designing Distributed Architectures](https://www.amazon.pl/Foundations-Scalable-Systems-Distributed-Architectures/dp/1098106067) It's on my list to read
+- [Cloud Native DevOps with Kubernetes: Building, Deploying, and Scaling Modern Applications in the Cloud](https://www.amazon.pl/Cloud-Native-DevOps-Kubernetes-Applications/dp/1098116828) Great book about k8s to understand core concepts.
+
+## Blogs
+List of People to follow recommended by Benjamin Rogojan, original link https://www.linkedin.com/feed/update/urn:li:activity:6979169467838652416?updateEntityUrn=urn%3Ali%3Afs_feedUpdate%3A%28V2%2Curn%3Ali%3Aactivity%3A6979169467838652416%29
+
+- [Chip Huyen](https://huyenchip.com/blog/)
```

Comment:
Please ensure the links are in `- [name](link) - description` format and we're good to merge!